addons/etcd/templates/cmpv.yaml

```diff
@@ -7,11 +7,11 @@ metadata:
 spec:
   compatibilityRules:
     - releases:
-        - v3.5.6
+        - {{ .Values.image.tag }}
```

Comment:
versions should be specified explicitly so that to support multiple versions.
```
  compatibilityRules:
    - compDef:  etcd-cmpd-v1
       releases:   v3.5.13
    - compDef:  etcd-cmpd-v2
       releases:   v3.5.6
  releases:
    - name: v3.5.6
       images: ...
    - name: v3.5.13
       images: ...

```

Reply:
solved, maintained two cmpd of etcd, busybox and distroless